### ISSUE TITLE
Add automated a11y testing

### ISSUE THREAD

**Issue by echappen** [cloud-gov/cg-ui#114]
Now that we're getting into building UI components, we should incorporate accessibility (a11y) testing into the repo. 

[pa11y-ci](https://www.npmjs.com/package/pa11y-ci) is the preferred method in the Engineering Practices guide, but I'm open to trying Chrome's [Lighthouse CI](https://www.npmjs.com/package/lighthouse-ci).

Every page/component tested should meet or exceed [WCAG 2.1](https://www.w3.org/TR/WCAG21/) AA standards.

### Acceptance criteria

- [ ] a check is added to GH actions for every pull request
- [ ] this check runs an a11y scanner tool on our components
- [ ] failure of this check blocks merging the PR

**Comment by echappen** [comment]
This will have the same limitations as with E2E testing (#347) because mock CF resources will need to be stood up and torn down during the CI runs.
